app/lib/ui/pr_card.dart

```diff
@@ -193,3 +184,31 @@ IconData iconForPRStatus(String status) {
       return FontAwesomeIcons.codeBranch;
   }
 }
+
+Widget conditionallyColoredDuration(BuildContext context, DateTime created) {
+  DateTime now = DateTime.now();
+  int ageInHours = now.difference(created).inHours;
+  Color color = Theme.of(context).colorScheme.rotten;
+
+  if (ageInHours < 24*14) {
+    color = Theme.of(context).colorScheme.stale;
+  }
+  
+  if (ageInHours < 24*5) {
+    color = Theme.of(context).colorScheme.waiting;
+  }
+
+  if (ageInHours < 8) {
+    color = Theme.of(context).colorScheme.fresh;
+  }
```

Comment:
I would propose the following:
```suggestion
  if (ageInHours < 24) {
    color = Theme.of(context).colorScheme.fresh;
  }
```
I wanna see some blue when I open the dashboard in the morning 🟦